bazel/repository_locations.bzl

```diff
@@ -688,9 +688,25 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         urls = ["https://github.com/llvm/llvm-project/releases/download/llvmorg-{version}/llvm-{version}.src.tar.xz"],
         release_date = "2020-03-23",
         use_category = ["dataplane_ext"],
-        extensions = ["envoy.wasm.runtime.wavm"],
+        extensions = [
+            "envoy.wasm.runtime.wamr",
+            "envoy.wasm.runtime.wavm",
+        ],
         cpe = "cpe:2.3:a:llvm:*:*",
     ),
+    com_github_wamr = dict(
+        project_name = "Webassembly Micro Runtime",
+        project_desc = "A standalone runtime with a small footprint for WebAssembly",
+        project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
+        version = "9710d9325f426121cc1f2c62386a71d0c022d613",
+        sha256 = "1d870f396bb6bdcb5c816326655b19a2877bbdf879255c335b8e84ce4ee37780",
+        strip_prefix = "wasm-micro-runtime-{version}",
+        urls = ["https://github.com/bytecodealliance/wasm-micro-runtime/archive/{version}.tar.gz"],
+        release_date = "2021-04-30",
```

Comment:
WAMR appears to have regular releases - https://github.com/bytecodealliance/wasm-micro-runtime/releases/tag/WAMR-04-15-2021. Can we move to a release vs. commit?

Reply:
The PR to support Envoy in WAMR hasn't been released. We use commit to utilize the latest updates.

Reply:
Since WAMR team still has several ongoing developments about *Envoy integration*. We'd like to keep using commits for agility.

Reply:
As per the dependency policy, releases are preferred over commits (particularly when there are regular releases) - https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md#declaring-external-dependencies. Once the WAMR Envoy integration stabilizes the plan should be to move to releases

Reply:
Ok. Got it. Thanks. We will move to release when WAMR releases in the future.